### ISSUE TITLE
Bookmark cells

### ISSUE THREAD

**Issue by conantroutman** [Elscrux/Creation-Companion#40]
Add a menu where you can bookmark commonly loaded cells, so you can quickly jump between them without having to scroll through hundreds of other cells each time. Would be a very useful feature for level designers.

Example scenario:
I'm working on a dungeon and I'm constantly back and forth between the dungeon cell and warehouse cells. If I could book mark the dungeon cell and the warehouse cells somewhere I could easily find them, it would save me a lot of time.

**Comment by Elscrux** [comment]
We can just open multiple render windows instead right?

**Comment by conantroutman** [comment]
> We can just open multiple render windows instead right?

I didn't know that feature was planned. Of course being able to load 2 cells at the same time is preferable.

**Comment by Elscrux** [comment]
It's not possible currently, but I'd like to have it in the future. I'll need to ask @MrTroble who is working on the renderer how feasible that is.

**Comment by MrTroble** [comment]
Yeah, I currently don't support multi viewport rendering although it is planned to include it in the future however the render
